chia/consensus/default_constants.py

```diff
@@ -54,6 +54,7 @@
     "MAX_GENERATOR_SIZE": 1000000,
     "MAX_GENERATOR_REF_LIST_SIZE": 512,  # Number of references allowed in the block generator ref list
     "POOL_SUB_SLOT_ITERS": 37600000000,  # iters limit * NUM_SPS
+    "SOFT_FORK_HEIGHT": 2000000,
```

Comment:
This is not a very forward-looking name. Maybe `SOFT_FORK_1_HEIGHT`?

Reply:
I'm hoping to delete this line in a few months.. thinking that we won't ever schedule more than one at a time